src/app/clusters/bindings/bindings.cpp

```diff
@@ -25,133 +25,63 @@
 #include <app-common/zap-generated/cluster-objects.h>
 #include <app/CommandHandler.h>
 #include <app/ConcreteCommandPath.h>
+#include <app/binding/BindingTable.h>
 #include <app/util/binding-table.h>
 #include <lib/support/logging/CHIPLogging.h>
 
 using namespace chip;
 using namespace chip::app::Clusters::Binding;
+using chip::app::BindingTable;
 
-EmberStatus prepareBinding(EmberBindingTableEntry & binding, NodeId nodeId, GroupId groupId, EndpointId endpointId,
-                           ClusterId clusterId)
-{
-    if (groupId && nodeId)
-    {
-        return EMBER_BAD_ARGUMENT;
-    }
-
-    binding.clusterId    = clusterId;
-    binding.local        = emberAfCurrentCommand()->apsFrame->destinationEndpoint;
-    binding.networkIndex = 0;
+namespace {
+// TODO: Add peer fabric index to command path
```

Comment:
Why are we hardcoding fabric indices at all?

Reply:
Currently the ZCL code does not pass the fabric in its message context. We know only the endpoint, cluster and the atttribute or command. Adding fabric in context will be in a future PR so as not to make this change too big.

Reply:
The `CommandHandler` knows the fabric and you have one at all the places where kPeerFabricId is being used.  I created https://github.com/project-chip/connectedhomeip/pull/12005 to put a nice API on it, but in the meantime `commandObj->GetExchangeContext()->GetSessionHandle().GetFabricIndex()` will work.